package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rewardops/sdk-node",
-  "version": "1.0.2",
+  "version": "1.1.1",
```

Comment:
Let's revert this line. It'll be updated in the next PR

Reply:
done